src/components/AddPlaidBankAccount.js

```diff
@@ -169,6 +187,7 @@ class AddPlaidBankAccount extends React.Component {
                         // User prematurely exited the Plaid flow
                         // eslint-disable-next-line react/jsx-props-no-multi-spaces
                         onExit={this.props.onExitPlaid}
+                        receivedRedirectUri={this.props.receivedRedirectURI}
```

Comment:
Why the difference in casing between the prop and the component's attribute ?

Reply:
Yeah, that's a funny point. Plaid named this config option as `receivedRedirectUri` but our style guide asks for acronyms like URI to be all capitalized.

Reply:
I can rename this prop here to also be `receivedRedirectURI`, which probably makes the most sense for consistency's sake.